### PR TITLE
fixes: particle network

### PR DESCRIPTION
**Pull request by techyNonso**
### Description

- [x] Resolves #71 

### Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [x] I have added suitable unit tests (if applicable).

To be reviewed by: @Nasfame

### CODE REVIEW

File: src/app/explore/page.tsx [hunk 1]
```diff
@@ -1,173 +1,85 @@
...
-        await create_user(user_data)
-        setUser(user_data)
-        push('/dashboard')
-      }
-    })
-  }
+  useEffect(() => {
+    checkStatus()
+    return
+  }, [])
```
[the remark this answers is not in the document]
Comment (Nasfame) [reply src/app/explore/page.tsx]: @techyNonso can u 
1. define checkStatus inside useEffect? 
2. or useCallback?

[the remark this answers is not in the document]
Comment (techyNonso) [reply src/app/explore/page.tsx]: useEffect cannot be async , I have fixed the initial error

File: src/app/explore/page.tsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ const Page = () => {
   useEffect(() => {
     checkStatus()
```
Comment (Nasfame) [inline src/app/explore/page.tsx:76]: add console.log("checkStatus")

**Review by Nasfame — CHANGES_REQUESTED** [review]
console.,log of no deps

**Review by Nasfame — APPROVED** [review]
LGTM, tested

**Pull request merged**